### PR TITLE
chore(stdlib): format the stdlib with the formatter

### PR DESCRIPTION
**Pull request by marcusroberts**
Applies the new formatting rules to the stdlib (pending some files with work underway and regex which needs more review

### CODE REVIEW

**Review by ospencer — APPROVED** [review]
Woohoo!

File: stdlib/runtime/numberUtils.gr [hunk 1]
```diff
@@ -1061,10 +1103,12 @@ let grisuRound = (buffer, len, delta, rest, ten_kappa, wp_w) => {
...
-    WasmI64.geU(WasmI64.sub(delta, rest), ten_kappa) && (
-      WasmI64.ltU(WasmI64.add(rest, ten_kappa), wp_w) ||
-      WasmI64.gtU(WasmI64.sub(wp_w, rest), WasmI64.sub(WasmI64.add(rest, ten_kappa), wp_w))
-    )
+    WasmI64.geU(WasmI64.sub(delta, rest), ten_kappa) &&
+    (WasmI64.ltU(WasmI64.add(rest, ten_kappa), wp_w) ||
+    WasmI64.gtU(
+      WasmI64.sub(wp_w, rest),
+      WasmI64.sub(WasmI64.add(rest, ten_kappa), wp_w)
+    ))
```
Comment (phated) [inline stdlib/runtime/numberUtils.gr:1111]: This is a little weird. We should look into it later.

File: stdlib/sys/file.gr [hunk 2]
```diff
@@ -818,7 +903,9 @@ export let rec fdStats = (fd: FileDescriptor) => {
 
     let flagsToWasmVal = (flag, i) => {
       let rightsInheriting = WasmI64.load(structPtr, 16n)
-      (rightsInheriting & (1N << WasmI64.extendI32U(WasmI32.fromGrain(i) >> 1n))) > 0N
+      (rightsInheriting &
+      1N << WasmI64.extendI32U(WasmI32.fromGrain(i) >> 1n)) >
+      0N
```
Comment (phated) [inline stdlib/sys/file.gr:908]: This one is interesting. Should we indent the line wrap since it is inside parens?

**Review by phated — APPROVED** [review]
This was a delight to review. Grainfmt is now producing code that'd I'd write by hand 🎉

**Pull request merged**